src/main/kotlin/fi/metatavu/lipsanen/projects/milestones/tasks/TaskController.kt

```diff
@@ -80,6 +81,13 @@ class TaskController {
             endDate = task.endDate,
             milestone = milestone,
             status = TaskStatus.NOT_STARTED,
+                // Todo: match types below
+            assignees = emptyList(),
+            userRole = task.userRole ?: UserRole.USER,
+            estimatedDuration = task.estimatedDuration ?: "",
+            estimatedReadiness = task.estimatedReadiness ?: "",
```

Comment:
```suggestion
            estimatedDuration = task.estimatedDuration,
            estimatedReadiness = task.estimatedReadiness,
```
If it is nullable, we don't need ?: "", it can be simply saved as null